### PR TITLE
DM-39989: Separate posargs from the tox command

### PR DESCRIPTION
**Pull request by rra**
tox v4 now requires that positional arguments to the underlying command be separated from the tox arguments with `--`. Add that marker before inputs.tox-posargs when invoking tox.

### CODE REVIEW

**Comment by rra** [comment]
I verified that tox is fine with using `--` as the last argument with nothing following it.

**Review by jonathansick — APPROVED** [review]
Good catch, thanks.

File: action.yaml [hunk 1]
```diff
@@ -67,4 +67,4 @@ runs:
     - name: Run tox
       shell: bash
       working-directory: ${{ inputs.working-directory }}
-      run: tox -e ${{ inputs.tox-envs }} ${{ inputs.tox-posargs }}
+      run: tox -e ${{ inputs.tox-envs }} -- ${{ inputs.tox-posargs }}
```
Comment (jonathansick) [inline action.yaml:70]: Oh, I haven't updated this for tox 4... should this be:

```suggestion
      run: tox run -e ${{ inputs.tox-envs }} -- ${{ inputs.tox-posargs }}
```

Comment (rra) [reply action.yaml:70]: Oh, indeed, good catch.

**Pull request merged**